src/17-infinite-discovery/02-artworks.ts

```diff
@@ -0,0 +1,126 @@
+import weaviate from "weaviate-ts-client"
+import dotenv from "dotenv"
+import { deleteIfExists } from "system/weaviate"
+import { ArtworksClassName } from "./types"
+
+dotenv.config()
+
+// Constants
+const CLASS_NAME: ArtworksClassName = "InfiniteDiscoveryArtworks"
+
+const client = weaviate.client({
+  host: process.env.WEAVIATE_URL!,
+})
+
+async function main() {
+  await prepareCollection()
+}
+
+main()
+
+async function prepareCollection() {
+  await deleteIfExists(CLASS_NAME)
+
+  const classWithProps = {
+    class: CLASS_NAME,
+    vectorizer: "text2vec-openai",
+    moduleConfig: {
+      "text2vec-openai": {
+        model: "text-embedding-3-small",
+        dimensions: 1536,
+        type: "text",
+        vectorizeClassName: false,
+      },
+    },
+    properties: [
+      {
+        name: "internalID",
+        dataType: ["text"],
+        moduleConfig: {
+          "text2vec-openai": {
+            skip: false,
```

Comment:
Tells `weaviate` to not include this property in the vector calculation. Its helpful for properties that are good for the querying client to have access to, but shouldn't be included in the vector calculation (i.e. they don't the model anything anything about the object its vectorizing).

Reply:
The way I wrote this was unclear. To clarify: `skip: true` means DON'T include in the vectorization and `skip: false` means DO include in the vectorization. We actually shouldn't include the internalID so I push up a fix on this.